src/poliastro/tests/tests_twobody/test_orbit.py

```diff
@@ -707,6 +707,57 @@ def test_perigee_and_apogee():
     assert_allclose(ss.r_p.to(u.km).value, expected_r_p.to(u.km).value)
 
 
+def test_expected_mean_anomaly():
+    # Example from Curtis
+    expected_mean_anomaly = 77.93 * u.deg
+
+    attractor = Earth
+
+    _a = 1.0 * u.deg  # Unused angle
+    a = 15300 * u.km
+    ecc = 0.37255 * u.one
+    nu = 120 * u.deg
+
+    orbit = Orbit.from_classical(attractor, a, ecc, _a, _a, _a, nu)
+    orbit_M = orbit.M
+
+    assert_allclose(orbit_M.value, expected_mean_anomaly.value, rtol=1e-2)
```

Comment:
There is an `assert_quantity_allclose` in Astropy that you can use https://astropy.readthedocs.io/en/stable/api/astropy.tests.helper.assert_quantity_allclose.html